mmengine/visualization/visualizer.py

```diff
@@ -412,26 +440,70 @@ def draw_points(self,
         assert positions.shape[-1] == 2, (
             'The shape of `positions` should be (N, 2), '
             f'but got {positions.shape}')
-        colors = color_val_matplotlib(colors)  # type: ignore
-        self.ax_save.scatter(
-            positions[:, 0], positions[:, 1], c=colors, s=sizes, marker=marker)
+        if self.backend == 'matplotlib':
+            if not (isinstance(marker, str) or marker is None):
+                raise TypeError(
+                    'The type of `marker` in `matplotlib` should be str, '
+                    f'but got {type(marker)}')
```

Comment:
marker can also be integer. See more details at https://matplotlib.org/stable/api/markers_api.html.

It would be helpful to also update the type hint and docstring.